File: integrations/copilot-sdk/typescript/examples/agentic_chat.ts
import type { CopilotClientPort } from "../dist/index.js";
import { defineAgent } from "./base.js";

export const createAgenticChatAgent = (client: CopilotClientPort) =>
  defineAgent(client, {
    agentId: "agentic_chat",
    description: "General purpose agentic chat assistant",
    instructions: "You are a helpful assistant. Use declared tools when requested.",
  });

File: integrations/copilot-sdk/python/examples/agents/agentic_chat.py
"""Agentic chat agent configuration."""

from typing import Any

from ag_ui_copilot_sdk import CopilotAgent

from .base import define_agent


def create_agentic_chat_agent(client: Any) -> CopilotAgent:
    return define_agent(
        client,
        name="agentic_chat",
        instructions="You are a helpful assistant. Use declared tools when requested.",
    )
